ui/raidboss/data/04-sb/alliance/royal_city_of_rabanastre.js

```diff
@@ -293,4 +293,461 @@
       },
     },
   ],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Added new combatant Archaeodemon': 'Added new combatant Archaeodemon', // FIXME
+        'Argath Thadalfus starts using Crippling Blow': 'Argath Thadalfus starts using Crippling Blow', // FIXME
```

Comment:
This seems incorrect.  I have plenty of logs if you need them to make better timelines from, but I would much rather have all of these be replaced with just `Argath Thadalfus` and to change all the syncs to be more properly something like `:SomeId:Argath Thadalfus` in the timeline sync.

Sorry for the bad timeline.  This was before better tools got made and I did this one by hand.

Reply:
I agree, but currently i am not that familiar with creating timelines (need to look at your guides for this)
Should i remove this one? or should it stay until better timelines are there?

Reply:
One option is to remove files that have this problem and follow up with other pull requests to make these timelines better.